geometry_msgs/Point pixel_lu
std_msgs/Int32 height
std_msgs/Int32 width
std_msgs/String tracker_name